data/json/items/tools.json

```diff
@@ -5209,7 +5209,7 @@
     "name": "mp3 player (on)",
     "name_plural": "mp3 players (on)",
     "description": "This mp3 player is turned on and playing some great tunes, raising your morale steadily while on your person.  It runs through batteries quickly; you can turn it off by using it.  It also obscures your hearing.",
-    "turns_per_charge": 150,
+    "turns_per_charge": 10000,
```

Comment:
This is a ludicrous increase in efficiency. I wonder if we'll see any bug reports about tools suddenly having infinite battery life?